components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/util/EndpointUtil.java

```diff
@@ -213,17 +216,17 @@ public static String getUserInfoClaimDialect() {
      * Returns the error page URL. If appName is not <code>null</code> it will be added as query parameter
      * to be displayed to the user. If redirect_uri is <code>null</code> the common error page URL will be returned.
      *
-     * @param errorCode
-     * @param errorMessage
-     * @param appName
-     * @return
+     * @param errorCode errorCode
```

Comment:
Shall we remove the duplicate param names here and in getLoginPageURL method comment ?